services/src/main/java/org/jboss/windup/web/services/model/RegisteredApplication.java

```diff
@@ -41,14 +43,14 @@
     private RegistrationType registrationType;
 
     @Column(length = 256)
-    @Size(min = 1, max = 256)
-    @NotNull
+    @NotBlankConstraint(message = "The application title must be set.")
+    @Size(min = 1, max = 256, message = "The application title must be set and not longer than 250 characters.")
     private String title;
 
     @Column(length = 2048)
-    @Size(min = 1, max = 2048)
-    @FileExistsConstraint
-    @NotNull
+    @NotBlankConstraint(message = "The application path must be set.")
+    @FileExistsConstraint(message = "The path does not exist on the server: ${validatedValue}")
+    @Size(min = 1, max = 2048, message = "The application title path must be set and not longer than 2048 characters.")
```

Comment:
The UI sent null when it was being created, and relied on the title being derived from the path on the server side. And then got message like "must not be null".
+1 to need localization, once we start doing it on the whole project.